db/beta_seed/user.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+user = User.new(
+  id_number:             11_111_111_111,
+  email:                 'cezmi@baum.omu.edu.tr',
+  password:              '123456',
+  password_confirmation: '123456',
+  activated:             true
+)
+
+User.skip_callback(:commit, :after, :build_address_information)
```

Comment:
@omu/backend Kullanıcı eklemek için bu callback'leri disable etmem gerekiyordu çünkü mernis, kps servisleri hata verecekti, dolayısıyla örnek kullanıcı eklenemeyecekti.

Araştırmalarıma göre bu işlem böyle yapılıyormuş. önce `skip_callback` ardından `User.after_create_commit` ile yeniden aktifleştir. Bu yöntem sizce de doğru mudur?

Reply:
Bildiğim bir konu değil maalesef, @isubas ?

Reply:
Konuyla ilgileniyorum, farklı bir yöntem deneyeceğiz son çare böyle devam ederiz.

Reply:
Identity ve adres oluşturma yapısından dolayı callback'leri atlamak gerekiyor. Bu şekilde devam ediyoruz.